### PR TITLE
CRE-587 Add Language Method Stubs

### PR DESCRIPTION
**Pull request by MarcFichtelVRC**
Follow up PR for https://github.com/vrchat/vrchat/pull/6877. Adds stubs for the new language methods to client sim so they can be tested in editor. The user can set the currently selected language in the client sim settings window.

### CODE REVIEW

File: Packages/com.vrchat.ClientSim/Editor/Windows/ClientSimSettingsWindow.cs [hunk 1]
```diff
@@ -382,6 +383,7 @@ private void DrawPlayerControllerSettings()
                 _settings.invertMouseLook = EditorGUILayout.Toggle(_invertMouseLookGuiContent, _settings.invertMouseLook);
                 _settings.playerHeight = EditorGUILayout.FloatField(_playerHeightGuiContent, _settings.playerHeight);
                 _settings.playerHeight = Mathf.Clamp(_settings.playerHeight, 0.2f, 80f); // TODO make consts for these.
+                _settings.currentLanguage = EditorGUILayout.TextField(_currentLanguageGuiContent, _settings.currentLanguage);
```
Comment (momo-the-monster) [inline Packages/com.vrchat.ClientSim/Editor/Windows/ClientSimSettingsWindow.cs:386]: Could we make this a dropdown field instead of write-in?

Comment (MarcFichtelVRC) [reply Packages/com.vrchat.ClientSim/Editor/Windows/ClientSimSettingsWindow.cs]: Good idea, done.
language-dropdown

**Review by momo-the-monster — APPROVED** [review]
Looks good, thanks for adding the dropdown.

**Comment by MarcFichtelVRC** [comment]
I have read the CLA Document and I hereby sign the CLA

**Pull request merged**